### DISCUSSION TITLE
I am having trouble with using Tailwind with Angular Library.

### DISCUSSION THREAD

**Discussion by ChetanSawant-97** [david-ui-org/david-ui-angular#19] [Q&A]
I am fairly new to Angular. I have a project I am working on I wanted to configure Tailwind with Angular Library. I tried a lot solution but I was not able to make it work. Please if you can you tell me how i can use tailwind with the library.

**Comment by draylegend** [comment]
@ChetanSawant-97 You can create a workspace with [nx.dev](https://nx.dev) and angular. You'll be asked whether you want to install tailwind or not. Then add david-ui-angular
